README.md

```diff
@@ -24,15 +24,15 @@ The latest version of Kitura-CredentialsHTTP requires **Swift 3.0**. You can dow
 ## API
 
 ### Basic authentication
-To create an instance of CredentialsHTTPBasic plugin, a `UserProfileLoader` function and an optional realm should be passed to the constructor:
+To create an instance of `CredentialsHTTPBasic` plugin, use the `verifyPassword` initializer that takes an optional realm:
 ```swift
```

Comment:
I don't like the description of the constructor as "the verifyPassword initializer". I prefer the previous version with `VerifyPassword` instead `UserProfileLoader`.